### ISSUE TITLE
Cover

### ISSUE THREAD

**Issue by gregoriopellegrino** [ways2read/InDesignA11Y#5]
Knowledge base reference: http://kb.daisy.org/publishing/docs/epub/cover.html

# Export setup
In the EPUB export wizard, in the General tab we set Cover as "Rasterise First Page" or "Choose Image" (and selecting an image file):
<img width="763" alt="Schermata 2022-06-29 alle 11 44 47" src="https://user-images.githubusercontent.com/5065125/176417943-364ca887-8501-4f56-af12-55033392dac5.png">
<img width="763" alt="Schermata 2022-06-29 alle 11 44 58" src="https://user-images.githubusercontent.com/5065125/176417965-533205ac-b354-4ea6-8fa9-f1795ceacaf0.png">

# Issue
With these export settings we get the `cover.xhtml` file containing the `<img>` tag with a non significant `alt` attribute:
 
` ` `
<img src="image/cover.png" alt="cover.png" style="max-width:100%;" />
 ` ` `

# How to fix it
Allow the user to set an appropriate alternative text for the cover image (maybe in the export wizard). Maybe placeholder based on metadata ` Cover of [title] by [author]`  may help the user.

# Other improvements
To improve the accessibility the cover page, you can enrich the semantics by using [DPUB Aria roles](https://www.w3.org/TR/dpub-aria-1.1/) in the HTML `role` attribute. In the file `3-enriched-semantics.epub` we have made a proposal for enriched semantics.

**Comment by CircularKen** [comment]
The default placeholder text could be derived from the Publication Lanaguge  - GitHub #6

**Comment by LauraB7** [comment]
User story: "As a screen reader user I can hear the relevant information included on the cover image."

**Comment by raman211** [comment]
Hi @gregoriopellegrino @ways2read @CircularKen @LauraB7,

This is Ramanjulu from Adobe,

Please confirm Whether the following proposed UX allows the content creator to add the required alternative text for the cover image only for the Reflowable workflow.
We are thinking of two possible designs (We would like to have your thoughts shared on these)

1) without the checkbox (means mandating the alternative text )
![AltTextFieldForCoverImage](https://github.com/ways2read/InDesignA11Y/assets/12660080/a738dd16-0df4-470a-b339-77f33e335b93)

2) With the checkbox, to allow the user to declare it as a decorative image or not
<img width="730" alt="Screenshot 2024-02-05 at 5 36 50 PM" src="https://github.com/ways2read/InDesignA11Y/assets/12660080/d0ed4c76-813d-4c4a-8dd0-bbf50c4386db">

We are thinking, that the option 2 should address the HTML role attribute case also.

Please also confirm if any changes are required for the Fixed Layout case.

Thanks,
Ramanjulu.

**Comment by LauraB7** [comment]
Hello @raman211. The second screenshot is the closest to the implementation that we'd like, with some caveats. 

- No checkbox, please. 
- The cover image will never need to be marked as decorative. 
- Suggested auto-populated text: "Cover: [title] by [author]. Description of image."

Additionally, it would be great if the resulting HTML was in a `<figure>` tag and had no inline styling (`style="max-width:100%;"`).

Finally, we would like time to test this before it's implemented to all InDesign users.

**Comment by raman211** [comment]
@LauraB7  Thanks a lot for the early feedback, we would love to share a pre-release build with the above suggestions incorporated.

**Comment by raman211** [comment]
Hi @gregoriopellegrino @ways2read @LauraB7 @CircularKen,

The feature is available in the latest InDesign PreRelease of 19.4
Looking forward to your valuable feedback, before releasing it in 19.4
Please share your feedback on it ASAP, so that we can accommodate the changes for the 19.4 release.

Thanks,
Ramanjulu.

**Comment by gregoriopellegrino** [comment]
Dear @raman211,
here is our feedback:

- the text put in the field for alt-text is not a placeholder, but is real text, so the user has to delete it before typing (not convenient)
   - If Indesign can only manage real text in the field (not a placeholder), the field should be left empty by default, as is the object export options alt text field
- the text you've put in the field for the alt-text as a placeholder is different from the text we suggested in previous comment https://github.com/ways2read/InDesignA11Y/issues/5#issuecomment-1929634949
- If the user does not enter anything in the field, the alt attribute should be left empty
   - right now it is exported "Cover Image"

**Comment by gregoriopellegrino** [comment]
Another thing (cited in comment https://github.com/ways2read/InDesignA11Y/issues/5#issue-1288530060): to improve the accessibility the cover page, you can enrich the semantics by using [DPUB Aria roles](https://www.w3.org/TR/dpub-aria-1.1/) in the HTML role attribute, in particular `role="doc-cover"` on the `<img>` tag.

**Comment by gregoriopellegrino** [comment]
Dear @raman211,
to prepare ourselves for the call scheduled on Monday, June 17th: are there any new updates or releases related to this issue (specifically https://github.com/ways2read/InDesignA11Y/issues/5#issuecomment-2025683720 and https://github.com/ways2read/InDesignA11Y/issues/5#issuecomment-2025848286)?

Currently, we are running the _InDesign Pre-Release 19.5 (Mainline CI #f95bd09)_ version on our machines.

Thanks
